test-app/tests/unit/flash/object-test.js

```diff
@@ -2,10 +2,56 @@ import { run, later } from '@ember/runloop';
 import { isPresent } from '@ember/utils';
 import { module, test } from 'qunit';
 import FlashMessage from 'ember-cli-flash/flash/object';
+import { disableTimeout, enableTimeout } from 'ember-cli-flash/test-support';
 
 const testTimerDuration = 50;
 let flash = null;
 
+module('FlashMessageObject disableTimeout', function (hooks) {
+  hooks.beforeEach(function () {
+    disableTimeout();
+    flash = FlashMessage.create({
```

Comment:
are you planning to remove the `EmberObject` usage before v5?

https://github.com/adopted-ember-addons/ember-cli-flash/blob/master/ember-cli-flash/src/flash/object.js#L2

Reply:
I wasn't specifically, although that will be a breaking change at some point...